ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -135,6 +135,7 @@ open class SearchBar: UIView {
         static let searchIconInset: CGFloat = 10.0
         static let searchTextFieldLeadingInset: CGFloat = 10.0
         static let searchTextFieldVerticalInset: CGFloat = 2
+        static let searchTextFieldInteractionMinWidth: CGFloat = 50
```

Comment:
where does this come from?

Reply:
actually it comes from my tests on DemoVC, we don't have spec from designer of this minimum width
if we need their confirm, who can I find?

Reply:
discussed with our team, we think this width is a protection when the text is too big or too long and let users to be able to interact. it not represented in design and doesn't have to be a precise value, and the `50` here is good enough for user to tap on textfield to edit.